site/src/components/Icons/EyeClosedIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import SvgIcon, { SvgIconProps } from "@mui/material/SvgIcon";
```

Comment:
Our codebase currently goes all in on MUI icons, and all of those should be available to import directly.
I would recommend looking at the [directory of icons](https://mui.com/material-ui/material-icons/?query=eye) and favoring those over custom components